style.css

```diff
@@ -10,7 +10,447 @@ body {
   padding: 0;
 }
 
+@media only screen and (min-width: 768px) {
+  header {
+    display: flex;
+    justify-content: space-between;
+    height: 10%;
+    align-items: center;
+    padding-left: 15%;
+    padding-right: 15%;
+    padding-top: 30px;
+    background-color: #fff;
+    width: 100%;
+  }
+
+  header a {
+    color: #6070ff;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    text-decoration: none;
+  }
+
+  .mobile_nav {
+    display: none;
+  }
+
+  .desktop_nav {
+    width: 35%;
+  }
+
+  .desktop_nav a {
+    margin-right: 10%;
+    font-family: "Poppins", sans-serif;
+    font-size: 15px;
+    font-weight: 500;
+    color: #344563;
+    text-align: center;
+  }
+
+  ul {
+    width: 20%;
+    display: flex;
+    justify-content: space-between;
+    padding: 0;
+  }
```

Comment:
- [ ] I have noticed that as I test for the responsiveness in the different desktop versions, the width of the `tech items` breaks and therefore destroys the UI. I suggest that you consider fixing the width so that the list items don't touch each other ☺️ .

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/220853004-e03a2e86-ca17-4b3f-a537-c745c7ef8f90.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/220853183-610c317d-a9be-44fb-9f16-ce3168792342.png)